src/Microsoft.AspNetCore.Razor.Tools/LazyStringWriter.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Razor.Tools
+{
+    internal class LazyStringWriter : TextWriter
```

Comment:
This seems like kinda a big/complicated hammer - do you dislike something about my suggestion to only log 100 errors from inside the tool?

Reply:
Nothing I dislike about that. But this is a more general fix that covers all the commands. Plus it also only creates the StringWriter when needed. It is actually not complicated at all. If you still feel strongly about this I can go back to your suggestion.